src/renderer/settings/components/NetworkSettings/NetworkSettings.js

```diff
@@ -121,8 +123,15 @@ export default class NetworkSettings extends React.PureComponent {
       return element.name === this.props.networkName;
     });
 
+    if (!this.props.networkName || !this.props.networkUrl) {
```

Comment:
Should we use `isEmpty` here too?

Reply:
Cool call xD 
Btw, I think we can ditch "withAlert" hoc, right? As it's completely replaced by the toasts. If so, I could include that aswell

Reply:
Let's keep it, because we will probably/possibly need alerts for things even if not the place we're using toasts.